test/suites/telemetry/telemetry.robot

```diff
@@ -122,15 +132,13 @@ Check MicroShift Metrics In Local Server    # robocop: disable=too-long-test-cas
 Setup
     [Documentation]    Test suite setup
     Check Required Env Variables
+    Check Required Proxy Variables
```

Comment:
Embed this one in `Check Required Telemetry Variables` and remove one keyword?

Reply:
fix it in https://github.com/openshift/microshift/commit/d712e5f1af81adb4021d2ba93407ae42b78cb200 commit